src/main/java/com/violeth/blockjs/blockjs/commands/BindCommand.java

```diff
@@ -0,0 +1,92 @@
+package com.violeth.blockjs.blockjs.commands;
+
+import com.violeth.blockjs.blockjs.BlockJS;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.TextColor;
+import org.bukkit.Material;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.io.File;
+import java.util.logging.Level;
+
+/** TODO: Deprecate event binding command and use run command on script with World.add[...]Listener instead? */
+public class BindCommand extends Command {
+    public BindCommand() {
+        super("bind");
+    }
+    @Override
+    public void handle(CommandSender sender, String[] args) {
+        if(sender instanceof Player) {
+            var player = (Player) sender;
+
+            if (args.length == 3) {
+                var rawScriptPath = args[2];
+
+                var scriptPath = new File(BlockJS.getScriptsFolder(), rawScriptPath);
+
+                if(!scriptPath.isFile()) {
```

Comment:
Server crashes if file doesn't exist, maybe add a check for existence or wrap in a try catch block to catch IOException?